package.json

```diff
@@ -173,7 +173,7 @@
         "ic-mops": "0.26.3",
```

Comment:
this is ancient

Reply:
That's a good point; thanks for noticing this. Updating this may break the extension (since Mops doesn't use semver), so I'll wait on this. This dependency is also only used for the "import Mops package" command.

Reply:
Sure, just do the safe thing!